CMakeLists.txt

```
@@ -13,14 +13,15 @@ set(CMAKE_CXX_FLAGS_DEBUG "-g")
 set(CMAKE_CXX_FLAGS_RELEASE "-O3")
 
 # Use standard c++20
-set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD 14)
```

```suggestion
set(CMAKE_CXX_STANDARD 20)
```
We'd better keep using std c++ 20, for older compiler, just change that manually.

---

accept